config/application-sample.yml

```diff
@@ -6,7 +6,7 @@ shopify_api_key: 'SHOPIFY_API_KEY'
 
 REDIS_URL: redis://localhost:6379/0/connector-shopify
 
-MNO_DEVPL_HOST: https://dev-platform.maestrano.io
+MNO_DEVPL_HOST: https://developer.maestrano.com
```

Comment:
Hi @alexnoox , in the last update @BrunoChauvet decided to use the production Sandbox, cause the environment is more stable. Let me know if this has changed and I will correct it!